packages/airnode-node/package.json

```diff
@@ -30,8 +30,10 @@
     "@api3/airnode-validator": "^0.6.0",
     "aws-sdk": "^2.992.0",
     "axios": "^0.27.2",
+    "axios-processing": "npm:axios@0.27.2",
```

Comment:
This is a way to install different versions of the same package. We pass `axios` and `ethers` to processing snippets and I find it useful to pin these versions explicitly. 

Alternatively, we can just use the versions we use internally. Those will get updated regularly by dependabot, which may break the processing users (unlikely, but it's possible).

Reply:
Not sure about that, maybe it will be easier to just drop this and use the versions we use in Airnode. Processing is a very advanced use case and people are expected to understand Airnode sources when using it.

Reply:
I think I personally lean towards just using the versions we use in Airnode

Reply:
I've dropped this commit. We will use the version that is bundled with Airnode.

Reply:
Burak's opinion is that we shouldn't expose third-party libraries at all, that it should be node modules and nothing else. The snippets I created recently only use built-in modules, so perhaps we should just remove axios and ethers.

Ultimately Processing's development flow should probably use something like webpack to create "static" minified snippets. WDYT?

Reply:
> Ultimately Processing's development flow should probably use something like webpack to create "static" minified snippets. WDYT?

I am not sure what you mean by this.

Reply:
> I am not sure what you mean by this.
It is possible to cherry pick out the code you need from libraries like ethers and include them in the snippet. This is a bit like binary compilation where you create a static executable (static executables bundle all dependencies, they don't dynamically load libraries at runtime).

> @aquarat are you sure this will not break your existing scripts? (e.g. oauth)
The snippets I created use the built-in Node modules :) this includes the OAuth and JWT snippets.

Reply:
Good point about just using the built in Node modules. It kinda feels like a slippery slope if we start allowing certain modules (ethers/axios), but not others (e.g. date-fns). Only allowing the built in modules feels like the better approach

Reply:
Reverted in https://github.com/api3dao/airnode/pull/1143/commits/23e3a8f7466517a10e264a8ab00f32216a11ceac

I am merging, but feel free to re-review.